.github/workflows/build/action.yml

```diff
@@ -35,4 +35,3 @@ runs:
         NEXT_PUBLIC_TENDERLY_SIMULATE_ENDPOINT_URL: ${{ fromJSON(inputs.secrets).NEXT_PUBLIC_TENDERLY_SIMULATE_ENDPOINT_URL }}
         NEXT_PUBLIC_WC_BRIDGE: ${{ fromJSON(inputs.secrets).NEXT_PUBLIC_WC_BRIDGE }}
         NEXT_PUBLIC_CYPRESS_MNEMONIC: ${{ inputs.e2e_mnemonic }}
-        NEXT_PUBLIC_SAFE_GELATO_RELAY_SERVICE_URL: ${{ fromJSON(inputs.secrets).NEXT_PUBLIC_SAFE_GELATO_RELAY_SERVICE_URL }}
```

Comment:
This is not needed as the constant has a fallback.

Reply:
Fallback is a fallback, but we need to pass both URLs from the secrets to the env.

Reply:
I've added both to the workflow.